src/userInterface/components/ConfigSelector/ConfigSelector.tsx

```diff
@@ -1,47 +1,31 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React from 'react';
 import classes from './ConfigSelector.module.scss';
 import { List } from 'antd';
-import { CogniteClient } from "@cognite/sdk";
 
-export const ConfigSelector: React.FC<{ sdk: CogniteClient, onClick: (item: any) => void }> = (props) => {
+export const ConfigSelector: React.FC<{
+    onClick: (configId: number) => void,
+    digitalTwinConfigMap: Map<number, unknown> | null,
+    selectedTwinId: number | null
+}> = (props) => {
 
-    const [digitalTwins, setDigitalTwins]= useState<any[]>([]);
-    const digitalTwinConfigMap = useRef<Map<string, unknown> | null>(null);
-    useEffect(()=>{
-        (async ()=> {
-            const response = await props.sdk.get(`${props.sdk.getBaseUrl()}/api/playground/projects/${props.sdk.project}/twins`);
-            if(response) {
-                const twins = response.data.data?.items;
-                const twinNames = [];
-                const twinMap = new Map();
+    const digitalTwins: { id: number, name: string }[] = [];
 
-                for(const twin of twins) {
-                    const twinName = twin?.data?.header?.name || twin.id;
-                    twinNames.push(twinName);
-                    twinMap.set(twinName, twin);
-                }
-                setDigitalTwins(twinNames);
-                digitalTwinConfigMap.current = twinMap;
-            }
-        })();
-    },[]);
+    props.digitalTwinConfigMap?.forEach((element: any) => {
```

Comment:
shall we remove references to digital twin. change all of them as json Configs.

Reply:
renaming all with 'json config'